mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -289,8 +287,11 @@ def message_auto_subscribe(self, updated_fields, values=None):
         """
         self.ensure_one()
 
-        # user_ids = [self.responsible_user_id._uid, self.manager_user_id._uid,
-        #            self.author_user_id._uid]
+        # user_ids = [
+        #    self.responsible_user_id._uid,
+        #    self.manager_user_id._uid,
+        #    self.author_user_id._uid,
+        # ]
```

Comment:
I left this code because because, i thouht it can be useful when we will find how  to automatically register those users as followers.